go.mod

```diff
@@ -52,13 +52,13 @@ require (
 	gorm.io/driver/mysql v1.1.1
 	gorm.io/gorm v1.21.12
 	helm.sh/helm/v3 v3.7.1
-	k8s.io/api v0.22.2
+	k8s.io/api v0.23.5
 	k8s.io/apiextensions-apiserver v0.22.1
```

Comment:
We might as well update this too just to keep things in sync as best we can.

Reply:
done!